modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/components/SimpleInputModal.es.js

```diff
@@ -147,9 +148,10 @@ const SimpleInputModal = ({
 								</span>
 							</label>
 
-							<input
+							<ClayInput
 								autoFocus
 								className="form-control"
+								component={isTextArea ? 'textarea' : 'input'}
```

Comment:
Nice choice, changing implementation to Clay! :+1: 

We can future-proof new `SimpleInputModal` API by just forwarding new prop to the [Clay component](https://github.com/liferay/clay/blob/master/packages/clay-form/src/Input.tsx#L162). Something like:
```js
<ClayInput
    component={mainFieldComponent}
>
```
New prop would be `mainFieldComponent`, to be consistent with existing naming pattern. In usages, you can set `mainFieldComponent` to `textarea`, if needed. I think you don't even need to set default here, because Clay [sets default to input](https://github.com/liferay/clay/blob/master/packages/clay-form/src/Input.tsx#L184), if `component` is unset.

Reply:
Oh I see, thank you